resources/lang/en/patrols/new_cat.json

```
@@ -4327,5 +4327,393 @@
+                        "mutual": true,
+                        "values": ["platonic", "trust", "comfort"],
+                        "amount": 15
+                    }
+                ],
+                "outsider_rep": 1
+            },
+            {
+                "text": "Oh wow, a kittypet! Sure, <i>technically</i> s_c should be sending the outsider away, but {PRONOUN/s_c/subject} just {VERB/s_c/have/has} so many questions! Luckily, the kittypet is willing to sate s_c's curiosity, and the Clancat eventually returns to camp with a new understanding, and potentially a new friend.",
+                "exp": 10,
+                "weight": 20,
+                "stat_trait": ["charismatic", "confident", "playful"],
+                "new_cat": [
+                    ["meeting", "kittypet", "age:adolescent"]
+                ],
+                "relationships": [
+                    {
+                        "cats_from": ["n_c:0"],
+                        "cats_to": ["patrol"],
+                        "mutual": true,
+                        "values": ["platonic", "trust", "comfort"],
+                        "amount": 15
+                    }
+                ],
+                "outsider_rep": 1
+            }
+        ],
+        "fail_outcomes": [
+            {
+                "text": "A loner? How interesting. p_l approaches the outsider but, before {PRONOUN/p_l/subject} can even meow out a greeting, the loner takes off, evidently startled by the c_n cat's appearance.",
+                "exp": 0,
+                "weight": 20,
+                "new_cat": [
+                    ["meeting", "loner", "age:adolescent"]
+                ],
+                "outsider_rep": 0
+            },
+            {
+                "text": "What in StarClan's name is a <i>kittypet</i> doing here? c_n's territory is no place for such a pampered creature, so p_l approaches the other young cat, intending to send {PRONOUN/n_c:0/object} back to wherever {PRONOUN/n_c:0/subject} came from. However, p_l's presence clearly intimidates the kittypet, who bolts away the moment {PRONOUN/n_c:0/subject} {VERB/n_c:0/see/sees} {PRONOUN/p_l/object}.",
+                "exp": 0,
+                "weight": 10,
+                "new_cat": [
+                    ["meeting", "kittypet", "age:adolescent"]
+                ],
+                "outsider_rep": 0
+            },
+            {
+                "text": "A loner? How interesting. p_l approaches the outsider but, before {PRONOUN/p_l/subject} can even meow out a greeting, the loner takes a swipe at {PRONOUN/p_l/object} and bolts away.",
+                "exp": 0,
+                "weight": 10,
+                "new_cat": [
+                    ["meeting", "loner", "age:adolescent"]
+                ],
+                "injury": [
+                    {
+                        "cats": ["patrol"],
+                        "injuries": ["claw-wound", "torn pelt", "torn ear", "minor_injury"],
```

```suggestion
                        "injuries": ["torn ear", "minor_injury"],
```
Those are pretty severe injury possibilities for a single swipe that is implied to be at a reasonably long distance.

---

I wrote this envisioning it as being similar to one of the fails for "gen_bord_lonerchase1", where one of the patrol cats gets a claw wound after being swiped at by a startled loner. Is there something I could add/remove/edit to make a claw wound a more realistic outcome in your eyes? /genq

---

If I remember correctly, the loner is busy fighting for their life from a dog in that patrol - their claws are already out and doing damage. There's a big difference in stakes:
 > "I was just fighting for my life with my claws out and biting at this big dog and ahhh! New threat! Diediedie!"

versus 

> "I was just chilling here and a strange cat walked up to me. I'm not chill with that so I'm gonna prove I'm not a pushover so they don't chase when I put some distance between us."
 
 This is just my opinion though!

---

Fair point, I see where you're coming from. I think I'll change this and the kittypet version to be just the minor stuff. I **_do_** still want the patrol to have a bit of ✨danger✨, so I think I'll whip up an outcome where the outsider is an aggressive rogue sometime in the future so that I can work-in the more serious injuries

---

Sounds good!